gm4_speed_paths/data/gm4_speed_paths/functions/main.mcfunction

```diff
@@ -1,4 +1,4 @@
 execute as @a[tag=!gm4_on_path] at @s if block ~ ~-0.9 ~ minecraft:dirt_path run function gm4_speed_paths:apply_effects/add_speed
```

Comment:
Since this is no longer the traditional main clock, add a header comment explaining this and where it is called from.

Additionally, remove it from the schedule loops in beet.yaml